spec/integration/fetching_content_item_spec.rb

```diff
@@ -126,7 +126,7 @@
   end
 
   context "a content item with a non-ASCII base_path" do
-    let(:content_item) { create(:content_item, base_path: URI.encode("/news/בוט לאינד")) }
```

Comment:
Thinking about these tests I'm not sure they're really testing the behaviour if we're just calling `Addressable::URI.encode`, it feels like we should hard code the URL encoded string. This would also allow us to find any potential problems in changing from `URI.escape` to `Addressable::URI.encode`.

Reply:
Sure, I think I understand your concern. I've added an extra commit - can you take another look?

Reply:
That seems like a strange rule about ASCII comments... but okay.

Reply:
Indeed. However, this is one of the very few instances where we have such characters. The advantage of this Cop is it stops people entering weird whitespace, (fancy) quote and even mathematical characters (in Smart Answers), which are all difficult to reproduce.

Reply:
Hold up - is that going to prevent emojis? 😿